### DISCUSSION TITLE
set_follow_target() not working for me

### DISCUSSION THREAD

**Discussion by itzjeppy** [ramokz/phantom-camera#301] [Question / Help Wanted]
[https://media.discordapp.net/attachments/610347507222052867/1240531335349534781/image.png?ex=6646e656&is=664594d6&hm=51f981a09ce39357e0798c897037d1d0a153f798a95c659bfd3a0b4bcd0d3e90&=&format=webp&quality=lossless&width=1245&height=701](url)

even after setting the follow target, the output console says none, and the camera isn't following my player

**Comment by ramokz** [comment]
If you call a print statement by passing the player as a parameter, does that return the player node?
I.e.
```gdscript
print(player)
```
